obal/data/playbooks/nightly/nightly.yaml

```diff
@@ -15,7 +15,7 @@
 
         - import_role:
             name: build_package
-      when: nightly_releaser is defined and '--arg jenkins_job=' in (build_package_tito_releaser_args | join(' '))
+      when: nightly_sourcefiles is not defined and nightly_githash is not defined
```

Comment:
Shouldn't this *still* check for `nightly_releaser` being defined? As otherwise the `set_fact` would bomb out?